app/assets/javascripts/discourse/app/components/bootstrap_mode_notice.js

```diff
@@ -0,0 +1,39 @@
+import discourseComputed from "discourse-common/utils/decorators";
+import Component from "@ember/component";
+import I18n from "I18n";
+import { alias } from "@ember/object/computed";
+import { htmlSafe } from "@ember/template";
+import { inject as service } from "@ember/service";
+
+export default Component.extend({
+  router: service(),
+  classNameBindings: ["hidden:hidden", ":bootstrap-notice"],
+
+  wizardRequired: alias("site.wizard_required"),
+  bootstrapModeEnabled: alias("siteSettings.bootstrap_mode_enabled"),
+  bootstrapModeMinUsers: alias("siteSettings.bootstrap_mode_min_users"),
+
+  @discourseComputed("bootstrapModeEnabled", "router.currentRouteName")
+  hidden(bootstrapModeEnabled, currentRouteName) {
+    const user = this.currentUser;
+    return !(
+      user &&
+      user.get("staff") &&
```

Comment:
You could probably do `this.currentUser?.staff` here (similar to the other PR).

Reply:
Sure, done.